static-site/src/pages/sars-cov-2.jsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import Helmet from "react-helmet";
+import {FaFile} from "react-icons/fa";
+import config from "../../data/SiteConfig";
+import NavBar from '../components/nav-bar';
+import MainLayout from "../components/layout";
+import UserDataWrapper from "../layouts/userDataWrapper";
+import { SmallSpacer, MediumSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import Footer from "../components/Footer";
+import allSARSCoV2Builds from "../../content/allSARS-CoV-2Builds.yaml";
+
+/*
+* This is a page to display all builds for SARS-CoV-2 in one place.
+* TODO:
+* - abstract 15px left margin
+* - tweak yaml design for builds list as necessary
+*/
+
+const buildComponent = (build) => (
+  <div key={build.url}>
+    <splashStyles.SitRepTitle >
+      <a href={build.url}>
+        <FaFile />
+        {` ${build.name} `}
+      </a>
+      (
+      {build.org.url === null ? build.org.name : <a href={build.org.url}>{build.org.name}</a>
+      }
+      )
+    </splashStyles.SitRepTitle>
+  </div>);
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      builds: allSARSCoV2Builds.builds,
```

Comment:
Is this data stored as `state` in preparation for a future where we may fetch the YAML rather than import it?

Reply:
Yes, but maybe we can just cross that bridge when we get there instead and leave it out of the component state.

Reply:
b7c79d6